backend/cmd/rewrite/main.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"bytes"
+	"github.com/gin-gonic/gin"
+	"golang.org/x/net/html"
+	"log"
+	"net/http"
+)
+
+func (rw *rewriter) getDefaultHtml() *html.Node {
+	req, _ := http.NewRequest("GET", rw.webEndpoint, nil)
+	res, _ := http.DefaultClient.Do(req)
+
+	doc, _ := html.Parse(res.Body)
+	return doc
```

Comment:
I'm skeptical here as well about completely hiding errors. Should probably at least log them IMO

Reply:
Initial thought was that this was a simple application made to fail or return a default whenever it failed, but yes, there's no reason not to log actual errors.